pkg/cosign/tuf/client.go

```diff
@@ -327,6 +297,10 @@ func rootCacheDir() string {
 	return rootDir
 }
 
+func cachedRemote(cacheRoot string) string {
+	return filepath.Join(cacheRoot, "remote.txt")
```

Comment:
Do you think we should write serialized structured data to this instead of just a string? Then we can easily append to it in the future.

Reply:
Yeah, I was kind of wondering whether we need something like a JSON blob here. I didn't want to overcomplicate it though. The only thing I could think of was multiple mirrors? WDYT?

Reply:
Yea, that would enable support for multiple roots of trust like I had mentioned here - https://github.com/sigstore/cosign/issues/1224#issuecomment-998161822

I think making the change to JSON with support for multiple mirrors (even if we just read only the first mirror for now, with a TODO for support for multiple mirrors in another PR) sounds good.